agreements/project-contribution/terms-and-conditions.md

```diff
@@ -223,6 +223,22 @@ While alignment with the Sponsor’s policies is encouraged where possible, thes
 
 Project leadership and operations naturally evolve due to changes in roles and responsibilities, or to unforeseen circumstances.
 
+Projects should designate at least one secondary contact to support continuity, but this may not always be possible at the time of initial signing.
+
+Where a project has a single contact, the Signatory and the Sponsor should work towards identifying additional contacts over time to support continuity.
+
+### Communication
+
+Signatories should be reachable and reasonably responsive to communication from the Sponsor regarding project activities, financial matters, and governance needs.
+
+**Annual contact review:** Projects must confirm or update their designated contacts annually through a verification process initiated by the Sponsor.
+
+- Where possible, this review will coincide with revew of [annual financial reports](#transparency-and-reporting).
+- If a project does not complete the verification within the designated timeframe, the Sponsor will attempt to re-establish contact.
```

Comment:
Is the designated timeframe defined in this doc or only in the verification process?

Reply:
Revised to link time periods together.